src/components/RegisterTab/RegisterTab.tsx

```diff
@@ -107,8 +110,11 @@ export class RegisterTab extends Component<RegisterTabProps, {personalEmailTouch
           isInvalid={!isPassValid(this.props.form.password)}
           placeholder='Password'
         />
-      </Form.Group>
-      <PasswordStrength password={this.props.form.password}/>
+        <Form.Text className='text-muted'>
+          We require a strong password for using our system.
```

Comment:
Instead of duplicating the text and style in two different places, why don't you create a component to hold the text and use the component both in the register tab and the reset password page.